generators/gulp/templates/gulpfile.js

```diff
@@ -1,12 +1,13 @@
+<% if (!babel) { -%>
```

Comment:
Actually no, gulpfile shouldn't be es2015 as it's the file building the babel code. We want this one compatible with node without precompilation.

So we can keep `'use strict';` as it's not a module file.